src/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/ParallelRangeManager.cs

```diff
@@ -37,6 +41,7 @@ internal struct IndexRange
     /// <summary>
     /// The RangeWorker struct wraps the state needed by a task that services the parallel loop
     /// </summary>
+    [StructLayout(LayoutKind.Auto)]
```

Comment:
(But generally, yes, it's something I typically do by default, when I remember and when it's not inappropriate for some reason, so that I don't need to worry about the size and order of the fields as much and can let the runtime take care of an optimal ordering for me.)